packages/block-library/src/columns/icons.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Can we remove this file now that the toolbar moved to components?

Reply:
Yes! Good spot.